typescript/internal/src/authManager.ts

```diff
@@ -1,12 +1,19 @@
 import clientMetadata from "./clientMetadata";
 import * as client from "openid-client";
+import { HabitatClient, HabitatAuthedAgentSession } from "./habitatClient";
+import { DidResolver } from "@atproto/identity";
+import { Agent } from '@atproto/api';
+
 
 const handleLocalStorageKey = "handle";
+const didLocalStorageKey = "did";
```

Comment:
just replace the handle it's used for the same thing. If the app wants to fetch the user profile data to pretty print they can do that separately.